frontend/src/metabase/search/components/SearchFilterModal/SearchFilterModal/SearchFilterModal.tsx

```diff
@@ -79,14 +98,14 @@ export const SearchFilterModal = ({
         </Flex>
         {availableFilters.map(key => {
           const Filter = filterMap[key];
-          return (
+          return Filter ? (
             <Filter
               key={key}
               data-testid={`${key}-search-filter`}
               value={output[key]}
               onChange={value => onOutputChange(key, value)}
             />
-          );
+          ) : null;
```

Comment:
tbh based on how you control `availableFilters` and `filterMap` yourself here. I'm not sure why we need to check if this is null. It feels to me `Filter` will never be `undefined`

Reply:
This will be resolved in the separate PR that I keep mentioning 😄